src/main/java/net/minecraftforge/fluids/FluidRegistry.java

```diff
@@ -356,6 +356,21 @@ public static String getDefaultFluidName(Fluid key)
         return name;
     }
 
+    @Nullable
+    public static String getModId(@Nullable FluidStack fluidStack)
+    {
+        if (fluidStack != null)
```

Comment:
I don't really see the benefit. Mojang has gone all-in with the null annotations instead of `Optional` style, so I'd rather stay close to that.